go.mod

```diff
@@ -3,42 +3,43 @@ module github.com/alphagov/paas-billing
 go 1.18
 
 require (
-	code.cloudfoundry.org/lager v0.0.0-20180322215153-25ee72f227fe
-	github.com/cloudfoundry-community/go-cfclient v0.0.0-20180727221732-7900c3c65270
+	code.cloudfoundry.org/lager v2.0.0+incompatible
+	github.com/cloudfoundry-community/go-cfclient v0.0.0-20220405200807-800807a1b060
 	github.com/cucumber/godog v0.12.2
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/gofrs/uuid v4.0.0+incompatible
 	github.com/labstack/echo v3.3.10+incompatible
-	github.com/labstack/gommon v0.2.9
-	github.com/lib/pq v0.0.0-20180327071824-d34b9ff171c2
+	github.com/labstack/gommon v0.3.1
+	github.com/lib/pq v1.10.5
 	github.com/onsi/ginkgo v1.6.0
```

Comment:
You're using Ginkgo 1.6.0 here, but `go install github.com/onsi/ginkgo/ginkgo` will install the version 2 CLI